shared/src/business/utilities/htmlGenerator/documents/order.scss

```diff
@@ -16,6 +16,14 @@
       line-height: 20px;
     }
 
+    ol ol {
+      list-style-type: lower-alpha;
+    }
+
+    // ul, ul ul {
```

Comment:
Can this be deleted?

Reply:
That....is not supposed to be commented, good catch